data/rhel6.cfg

```diff
@@ -46,6 +46,8 @@ logging --level=info
 poweroff
 # System timezone
 timezone  America/New_York
+# Skip Unsupported Hardware warning
```

Comment:
Can you add this BZ here too for future lookup: https://bugzilla.redhat.com/show_bug.cgi?id=824963